src/components/RadioButton.js

```diff
@@ -1,38 +1,88 @@
+// ***NEED TO FIGURE OUT HOW TO SWITCH ON AND OFF AFTER SELECTION***
```

Comment:
Not sure what happened here.  Try see if you can reset https://raw.githubusercontent.com/DaDataDudes/react-native-form/development/src/components/RadioButton.js